tkupchyn/homework_03/consecutive_strings.py

```diff
@@ -0,0 +1,17 @@
+
+def consecutive_concatenation(strings: list, k: int) -> str:
+    """
+    Return the first longest string consisting of k consecutive strings taken in the array.
+
+    Args:
+        strings (list): list of strings
+        k (int): number of consecutive strings which need to be concatenated
+    Returns:
+        str: longest string consisting of k consecutive strings
+    """
+    concatenated_strings = []
+
+    for i in range(0, len(strings) - k + 1):
+        concatenated_strings.append(''.join((strings[i: i + k])))
```

Comment:
if concatenated_string is a generator expression you will avoid creating intermediate list

Reply:
changed to generator expression